packages/chatbot-server-mongodb-public/src/processors/makeNShotUserMessageFunction.ts

```diff
@@ -0,0 +1,88 @@
+import { Message } from "mongodb-chatbot-server";
+import { ZodObject, ZodRawShape } from "zod";
+import { stripIndents } from "common-tags";
+import { zodToJsonSchema } from "zod-to-json-schema";
+import { OpenAI } from "openai";
+import {
+  ChatCompletionMessageParam,
+  ChatCompletionTool,
+} from "openai/resources";
+
+export interface MakeNShotUserMessageExtractorFunctionParams {
```

Comment:
```suggestion
export interface MakeFewShotUserMessageExtractorFunctionParams {
```

Is "few shot" easier to understand than "N shot" especially given we have `fewShotExamples` as a key?

Reply:
makes sense to me. updating all naming